packages/create-expo/src/cli.ts

```diff
@@ -38,7 +38,7 @@ async function run() {
       [
         `-y, --yes             Use the default options for creating a project`,
         `    --no-install      Skip installing npm packages or CocoaPods`,
-        chalk`-t, --template {gray [pkg]}  NPM template to use: blank, tabs, bare-minimum. Default: blank`,
+        chalk`-t, --template {gray [pkg]}  NPM template to use: default, blank, tabs, bare-minimum.`,
```

Comment:
@brentvatne maybe we should omit this, we also support github URLs or tarbal paths with `--template`. That's not reflected here at all either.

Reply:
Wouldn't the more appropriate solution be to include all options here?